Resources/Prototypes/Entities/Objects/Specific/revolutionary.yml

```
@@ -0,0 +1,27 @@
+- type: entity
+  name: revolutionary manifesto
+  parent: BaseItem
+  id: RevolutionaryManifesto
+  description: A manifesto written by a self-proclaimed revolutionary leader. Very critical of Nanotrasen.
+  components:
+    - type: Sprite
+      sprite: Objects/Misc/books.rsi
+      layers:
+      - state: paper
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)